websites/F/Free Web Novel/dist/metadata.json

```diff
@@ -0,0 +1,94 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "BigBoi",
+		"id": "853932304257908746"
+	},
+	"service": "Free Web Novel",
+	"description": {
+		"en": "We are offering free books online read! Read novel updated daily: light novel translations, web novel, chinese novel, english Novel, japanese novel and korean novel online."
+	},
+	"url": "freewebnovel.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/ePGzX1S.png",
+	"thumbnail": "https://i.imgur.com/PJw9FKX.png",
```

Comment:
if you could find a taller picture (closer to 16:9) that would be awesome

Reply:
The thumbnail?